src/main/java/core/basesyntax/shape/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class Circle extends Shape {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * this.radius * this.radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append("Shape: Circle");
+        stringBuilder.append(", area: ").append(this.getArea()).append(" sq. units");
+        stringBuilder.append(", radius: ").append(this.radius).append(" units");
+        stringBuilder.append(", color: ").append(this.getStringColor());
+
+        System.out.println(stringBuilder);
```

Comment:
Instead of using 'System.out.println()', consider returning the string and printing it outside the 'draw()' method to follow the Single Responsibility Principle.